### PR TITLE
Trust delay based BWE

### PR DESCRIPTION
**Pull request by grgustaf**
(Patch from Sriranjan Srikantam.)

Signed-off-by: Geoff Gustafson <geoff@linux.intel.com>

### CODE REVIEW

**Comment by grgustaf** [comment]
@taste1981, should this be wrapped in INTEL_GPRA or is it also sensible for the original delay-based BWE?

**Comment by taste1981** [comment]
> @taste1981, should this be wrapped in INTEL_GPRA or is it also sensible for the original delay-based BWE?

You don't need to wrap it under INTEL_GPRA logic.

File: modules/bitrate_controller/send_side_bandwidth_estimation.cc [hunk 1]
```diff
@@ -422,7 +422,10 @@ void SendSideBandwidthEstimation::CapBitrateToThresholds(int64_t now_ms,
   if (bwe_incoming_ > 0 && bitrate_bps > bwe_incoming_) {
     bitrate_bps = bwe_incoming_;
   }
-  if (delay_based_bitrate_bps_ > 0 && bitrate_bps > delay_based_bitrate_bps_) {
+  if (delay_weight == 100) {
+    bitrate_bps = delay_based_bitrate_bps_;
+  }
+  else if (delay_based_bitrate_bps_ > 0 && bitrate_bps > delay_based_bitrate_bps_) {
```
Comment (taste1981) [inline modules/bitrate_controller/send_side_bandwidth_estimation.cc]: format the code to put else if right after } above.

Comment (grgustaf) [reply modules/bitrate_controller/send_side_bandwidth_estimation.cc]: I think it's already formatted the way you're requesting:

https://github.com/open-webrtc-toolkit/owt-deps-webrtc/blob/db0c98321646b0463e809423499f547be0eb1f62/modules/bitrate_controller/send_side_bandwidth_estimation.cc#L422-L430

Comment (grgustaf) [reply modules/bitrate_controller/send_side_bandwidth_estimation.cc]: Oh I see what you mean.

**Review by taste1981 — APPROVED** [review]

**Pull request merged**